src/main/java/uk/gov/pay/directdebit/partnerapp/model/GoCardlessAppConnectTokenEntity.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.pay.directdebit.partnerapp.model;
+
+import uk.gov.pay.directdebit.common.util.RandomIdGenerator;
+
+public class GoCardlessAppConnectTokenEntity {
```

Comment:
Hmm… I’m not keen on `Token` because of the confusion with access token.

In addition, this class has more than just a token: it has a `gatewayAccountId` and `redirectUri` etc. I think there is an argument that this could be called a `GoCardlessAppConnectAccountEntity` or something.

Reply:
`GoCardlessAppConnectAccountEntity` 👍